assets/css/style.css

```diff
@@ -510,6 +509,47 @@ html[data-current="contact"] main > form button {
     }
 }
 
+hr {
+    height: 1px;
+    opacity: 0.8;
+    padding-bottom: 100px;
+}
+
+blockquote{
+  font-size: 1em;
+  width:100%;
+  margin:50px auto;
+  font-family:Open Sans;
+  font-style:italic;
+  color: #555555;
+  padding:1em;
+  border-left:8px solid #DE3545 ;
+  line-height:1.6;
+  position: relative;
+  background:#EDEDED;
+}
+
+blockquote::before{
+  font-family:Arial;

```

Comment:
What does setting the font here do?

I like the new look of `blockquote`, it’s great on a blog post:

![image](https://user-images.githubusercontent.com/43280/98839010-9d2a4c80-240a-11eb-82cf-81327cea2848.png)

But maybe needs tweaking on a page?

![image](https://user-images.githubusercontent.com/43280/98838940-897ee600-240a-11eb-95b6-b3215162665a.png)

Reply:
Looks like the font setting does nothing. Removed. Will need to tweak block quotes on pages in a future pr. Thanks.